content/config_rb_client.md

```diff
@@ -444,13 +446,16 @@ This configuration file has the following settings:
 
 `minimal_ohai`
 
-:   Run the Ohai plugins for name detection and resource/provider
-    selection and no other Ohai plugins. Set to `true` during
-    integration testing to speed up test cycles.
+:   Run a minimal set of Ohai plugins providing data necessary for the
+    execution of Chef Infra Client's built-in resources. Setting this
+    to true will skip many large and time consuming plugins such as
+    `cloud` or `packages`. Setting this to true may break cookbooks
+    that assume all Ohai data will be present.
```

Comment:
Runs only the Ohai plugins that determine the O/S platform, hostname, and init system.  This can be used to speed up testing or `chef-apply` scripts, but will break many cookbooks which expect more extended Ohai attributes.

Reply:
That's all there. People don't need to know about chef testing here.